src/inputs/datePickerInput/datePickerInput.jsx

```diff
@@ -264,8 +271,13 @@ class DatePickerInput extends React.PureComponent {
         }
     }
 
+    onInputClick() {
+        this.setState({ isFocused: true });
+    }
+
     onInputFocus() {
         this.setOpen(true);
+        this.setState({ isFocused: true });
```

Comment:
Why are we setting `isFocused` to `true` and above in the `onInputClick()` method?

Reply:
So, we click on control (or control got focus) and able to edit the text - in this case we set the `isFocused` flag and do not use the `displayFormat`.